### ISSUE TITLE
0 boxes all the time when predicting

### ISSUE THREAD

**Issue by fakhouri-junior** [experiencor/basic-yolo-keras#111]
Hi I trained the network (tiny yolo) per instructions given in read-me but when I do prediction i get 0 boxes all the time even on training data. I am using raccoon dataset, and after training I saw the loss was about 0.0###, so I am not sure what's the problem.

Thanks a lot

**Comment by usatenko** [comment]
I also got very bad results on tiny yolo. Can you try to train on full yolo with pretrained weights added `"pretrained_weights":   "",`

**Comment by dikshant2210** [comment]
Did you try playing with threshold values?
It helped me

**Comment by fakhouri-junior** [comment]
@dikshant2210 @usatenko Sorry for late reply, No I actually have not yet tried, but will do and update the post. Thank you for your help

**Comment by kodlan** [comment]
@dikshant2210 what values do you mean? "object_scale", "no_object_scale", "coord_scale", "class_scale" ?

**Comment by yuye1992** [comment]
Did you solve it? I also got the same problem.

**Comment by kodlan** [comment]
@yuye1992 nope. I've tried a changing those values and that didn't help.
Also tried different backends (except full yolo cause it takes looong time to train) with no luck.

Will try full yolo.. :(

**Comment by kodlan** [comment]
Full yolo is also not working :(

**Comment by experiencor** [comment]
@kodlan Did you do warm-up training? It's critical. Refer to the readme for other tips.

**Comment by kodlan** [comment]
@experiencor I've tried with warm-up and without, not working in both cases for your kangaroo dataset :(

**Comment by prantiksen4** [comment]
How to set the threshold value for confidence?

**Comment by experiencor** [comment]
@prantiksen4 Please refer to ```frontend.py```.
